lib/ruby_http_client.rb

```diff
@@ -19,6 +19,7 @@ def initialize(response)
     # Returns the body as a hash
     #
     def parsed_body
+      return @body if !@body
```

Comment:
this method is expected to return a `Hash`, you're going to return `nil`.  Would it be better to raise an exception and handle it properly?  If not, return a `nil`

```ruby
return nil if @body.nil?
```